src/renderer/prefComponents/imageUploader/index.vue

```diff
@@ -152,8 +180,21 @@ export default {
   & .el-input {
     max-width: 242px;
   }
+  & .el-button.btn-reset,
   & .button-group {
     margin-top: 30px;
   }
+  & .pref-cb-legal-notices {
```

Comment:
Shall we put this style in  
>src/renderer/prefComponents/imageUploader/legalNoticesCheckbox.vue

Reply:
I don't think so because this are elements in the current file and should not included in a general element.